app/scripts/dimApp.config.js

```diff
@@ -49,6 +49,15 @@
           SyncService.init();
         };
 
+        var chromeVersion = /Chrome\/(\d+)/.exec($window.navigator.userAgent);
+        if (chromeVersion && chromeVersion.length === 2 && parseInt(chromeVersion[1], 10) < 50) {
```

Comment:
fair enough. it's not like someone with chrome 44 would only be able to update to chrome 48 or something anyway, they'd bump to latest. just getting the message out there to update should be enough motivation.